lib/datadog/core/telemetry/schemas/events/v1/app_started.rb

```diff
@@ -0,0 +1,23 @@
+module Datadog
+  module Core
+    module Telemetry
+      module Schemas
+        module Events
+          module V1
+            # Describes payload for telemetry V1 API app_started event
+            class AppStarted
+              attr_reader :configuration, :dependencies, :integrations, :additional_payload
+
+              def initialize(configuration = nil, dependencies = nil, integrations = nil, additional_payload = nil)
```

Comment:
I'm wondering if we should make some of these args required.

The general idea is we don't want poorly formed events; if you can build an event object, then its legit, so we don't have to validate it downstream. So these constructors should ensure the input is good.

This means we likely need validation logic implemented, as well as corresponding tests.